### PR TITLE
P0 cards input

### PR DESCRIPTION
**Pull request by Muzammil-khan710**
Added card and input components

### CODE REVIEW

File: docs/components/cards/cards.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
+.card-img {
+    height: 200px;
```
Comment (DivyanshSareen) [inline docs/components/cards/cards.css:2]: use relative values

File: docs/components/cards/cards.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    width: 100%;
+    border-radius: 5px 5px 0 0 ;
+}
+
+.card {
+    border: 1px solid black;
+    /* max-width: 600px; */
```
Comment (DivyanshSareen) [inline docs/components/cards/cards.css:9]: remove comment if not for future use

File: docs/components/cards/cards.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+.horizontal-card {
+    display: flex;
+    max-width: 600px;
+}
+
+.horizontal-card-img {
+    height: 200px;
+    border-radius: 5px 0 0 5px;
```
Comment (DivyanshSareen) [inline docs/components/cards/cards.css:65]: relative units here

**Review by DivyanshSareen — APPROVED** [review]
Good to merge

File: docs/components/cards/cards.css [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+.overlay-txt {
+    position: absolute;
+    top: 5rem;
+    right: 1.5rem;
+}
+
+.horizontal-card {
+    display: flex;
+    max-width: 600px;
```
Comment (rutu14) [inline docs/components/cards/cards.css:60]: Unit - relative units can be applied.

**Review by rutu14 — APPROVED** [review]
UI Suggestion : The cross icon on cards can have a bigger font size or some color change to darker shade.

**Review by GarvG — APPROVED** [review]
After Doing the necessary changes looks good to merge

**Pull request merged**